test/contracts-network/colony-expenditure.js

```diff
@@ -219,47 +265,61 @@ contract("Colony Expenditure", (accounts) => {
       totalPayout = await colony.getFundingPotPayout(expenditure.fundingPotId, token.address);
       expect(totalPayout).to.eq.BN(WAD);
     });
+  });
 
-    it("should allow arbitration users to set the payoutModifier", async () => {
-      await checkErrorRevert(
-        colony.setExpenditurePayoutModifier(1, UINT256_MAX, expenditureId, SLOT0, WAD.divn(2), { from: ADMIN }),
-        "ds-auth-unauthorized"
-      );
+  describe("when locking expenditures", () => {
+    let expenditureId;
 
-      await colony.setExpenditurePayoutModifier(1, UINT256_MAX, expenditureId, SLOT0, WAD.divn(2));
+    beforeEach(async () => {
+      await colony.makeExpenditure(1, UINT256_MAX, 1, { from: ADMIN });
+      expenditureId = await colony.getExpenditureCount();
+    });
+
+    it("should allow owners to lock expenditures", async () => {
+      let expenditure = await colony.getExpenditure(expenditureId);
+      expect(expenditure.status).to.eq.BN(DRAFT);
+
+      await checkErrorRevert(colony.lockExpenditure(expenditureId, { from: USER }), "colony-expenditure-not-owner");
+      await colony.lockExpenditure(expenditureId, { from: ADMIN });
 
-      const expenditureSlot = await colony.getExpenditureSlot(expenditureId, SLOT0);
-      expect(expenditureSlot.payoutModifier).to.eq.BN(WAD.divn(2));
+      expenditure = await colony.getExpenditure(expenditureId);
+      expect(expenditure.status).to.eq.BN(LOCKED);
     });
 
-    it("should not allow arbitration users to set the payoutModifier above the maximum", async () => {
-      await colony.setExpenditurePayoutModifier(1, UINT256_MAX, expenditureId, SLOT0, MAX_PAYOUT_MODIFIER);
+    it("should not allow the owner to cancel the expenditure", async () => {
+      await colony.finalizeExpenditure(expenditureId, { from: ADMIN });
```

Comment:
This should check the owner can't cancel it when it's locked, not when it's finalized.